release notes/v0.55.2.md

```diff
@@ -0,0 +1,5 @@
+k6 `v0.55.2` is a patch release that fixes packaging issue:
+
+- Fixing a packaging problems in `v0.55.1`
```

Comment:
Sorry :see_no_evil: that's the last one, I believe that version is better from clarity and structure

```suggestion
k6 `v0.55.2` is a patch release that fixes packaging issue.
```